app/actions/websocket/users.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {type Model} from '@nozbe/watermelondb';
```

Comment:
why this change?

Reply:
I believe it was moved by the linter's autofix, no personal reason 🤷🏼 
At some point between changes I imported something else from it and `type` was moved inside, which caused a whole reordering. I'll move `type` outside of the brackets so it goes back to its previous place